### ISSUE TITLE
Made up LaTeX2e/AMS Mathjax aliases/symbols

### ISSUE THREAD

**Issue by jmeas** [Khan/KaTeX#130]
Mathjax makes up some aliases for symbols, and at times invents its own symbols. Including them makes KaTeX more consistent with Mathjax, but less consistent with LaTeX.
###### LaTeX2e Symbols/Aliases

`\omicron` - ο (distinct from the o on your keyboard)
`\iint` - made up; a useful addition ∬
`\iiint` - made up; a useful addition ∭
`lt` - Alias of <
`gt` - Alias of >
`notChar` - Alias of /
`dotsc` - Alias of \ldots
`dotso` - Alias of \ldots
`And` - Alias of $
`dotsb` - Alias of \cdots
`dotsm` - Alias of \cdots
`dotsi` - Alias of \cdots
# 
###### AMS Symbols/Aliases

These are quick references to italicized versions of Greek capital letters.

`\\varGamma`
`\\varDelta`
`\\varTheta`
`\\varLambda`
`\\varXi`
`\\varPi`
`\\varSigma`
`\\varUpsilon`
`\\varPhi`
`\\varPsi`
`\\varOmega`

_Tested against `LaTeX2e` with the `amssymb` package._

For now, these will be omitted from the big symbols PR due to the fact that they aren't standard and need clarification; they can be added later, if we choose, in a separate commit.

What would you like to do @xymostech / @spicyj?

**Comment by RoyiAvital** [comment]
Hi,
Will it support more complex things like:

\begin{align}
[[ The Math ]]
\end{align}

It would be amazing!

**Comment by xymostech** [comment]
@RoyiAvital the `align` environment is unrelated, you're looking for issue #61. Yes, we will support that in the future.

**Comment by kevinbarabash** [comment]
I think we have most of the "LaTeX2e Symbols/Aliases" listed above with the exception of `notChar`.  The `AMS Symbols/Aliases` will require new fonts which is blocked on #816.

**Comment by kevinbarabash** [comment]
This is no longer block as #816 has been completed.

**Comment by akalin** [comment]
The dots macros aren't implemented yet, right? Are there any plans for those?

**Comment by kevinbarabash** [comment]
@akalin they were implemented by @edemaine in #794.  I believe that `\varGamma` et al is the only remaining work in this issue.

**Comment by kevinbarabash** [comment]
I think it should be pretty easy to implement these aliases, e.g.
```
defineMacro("\\varGamma", "\\mathit{\\Gamma");
```

**Comment by kevinbarabash** [comment]
Superseded by #1077.
